ff-suggest-two-channels/index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Suggest Demo</title>
```

Comment:
Can I interest you in one of these titles?
```suggestion
    <title>Two Channel Suggest Demo</title>
```
```suggestion
    <title>Suggest With Two Channels Demo</title>
```